src/Umbraco.Web/PublishedCache/NuCache/PublishedSnapshotService.cs

```diff
@@ -140,8 +142,19 @@ internal class PublishedSnapshotService : PublishedSnapshotServiceBase
                         lock (_storesLock)
                         {
                             _contentStore?.ReleaseLocalDb(); //null check because we could shut down before being assigned
+                            // Make sure the local dbs are disposed of if the content store was not created.
+                            try
+                            {
+                                _localContentDb?.Dispose();
```

Comment:
I know you've added additional try/catch's here, but my first comment was saying that you don't need to dispose of this here at all since it's definitely going to be disposed of in `_contentStore?.ReleaseLocalDb` above.

Reply:
Your call - just noticed the use of ?. and catering for the null scenario.

Reply:
I suppose it's safer, fine to leave the additional disposal call